app/scripts/directives/processTemplateDialog.js

```diff
@@ -208,6 +208,14 @@
       }));
     };
 
+    // TODO: Make this as a reusable component since we are using similar method also in origin-web-catalog
```

Comment:
I think the TODO is really to reuse the select-project component. We'd need to update the component to optionally disable creating new projects, however. Do you mind updating the comment?